packages/ui-components/src/lib/Library/index.js

```diff
@@ -125,8 +145,19 @@ const LibraryComponent = (props) => {
       filtered = filtered.filter(item => currentFilter.indexOf(item.itemType) > -1);
     }
 
+    if (ciceroVersion) {
+      // pre-existinmg length for filtered present
+      if (filtered.length >= 1) {
+        filtered = filtered.filter((item) => item.version === ciceroVersion);
+      } else {
+        // else, populate filtered with available options again
+        filtered = filterItemsByQuery();
+        filtered = filtered.filter((item) => item.version === ciceroVersion);
+      }
```

Comment:
Could this be written another way?

```js
      if (filtered.length < 1) {
        // populate filtered with available options again
        filtered = filterItemsByQuery();
      } 
      filtered = filtered.filter((item) => item.version === ciceroVersion);
```

Reply:
Ok sure, let me test this out and will make a pr.

Reply:
Made the requested change in yesterday's pr